gratipay/security/__init__.py

```diff
@@ -54,7 +54,6 @@ def add_headers_to_response(response):
             "img-src *;"
             "font-src 'self' assets.gratipay.com cloud.typography.com data:;"
             "upgrade-insecure-requests;"
-            "block-all-mixed-content;"
```

Comment:
Good question. I guess the latter gives us the flexibility to serve http on localhost, ya?